File: tests/nonsmoke/functional/CompileTests/OpenMP_tests/fortran/parallel.f
C***************************************************
C test the simplest helloworld in OpenMP Fortan
C comments are attached to real statements
C Liao, 9/30/2010
C***************************************************
      PROGRAM HELLO
!$OMP PARALLEL
      PRINT *, 'Hello World!'
c$omp end parallel 
      PRINT *, 'the end'
      END

File: tests/nonsmoke/functional/CompileTests/OpenMP_tests/parallel.c
#include<assert.h> 

int main(void)
{
  int i =100;
  int j = 77;
#pragma omp parallel firstprivate(i)
  {
    assert(i == 100);
    assert(j == 77);
  }
#pragma omp parallel private(i)
  {
    assert(i != 100);
  }

  return 0;
}
